<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScreen</class>
 <widget class="QWidget" name="GameScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>693</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton {
    background-color: lightblue;
    color: white;
    font-size: 18px;
    font-weight: bold;
    border: none;
    border-radius: 12px;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #357ABD;
}

QPushButton:pressed {
    background-color: #2D5D9F;
}


QLabel {
    color: #1A1A1A;
    font-size: 22px;
    font-weight: semi-bold;
}


QGridLayout {
    spacing: 5px;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="gridWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>675</width>
       <height>675</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="time">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    color: #2c3e50;
}</string>
        </property>
        <property name="text">
         <string>Time: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>7000</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="homeReturn">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="showMistakes">
        <property name="maximumSize">
         <size>
          <width>3000</width>
          <height>2000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    color: #e0e0e0;
    background-color: #2b2f4a;         
    font-size: 18px;
    font-weight: 800;
    font-family: &quot;Orbitron&quot;, &quot;Poppins&quot;, &quot;Segoe UI&quot;, &quot;Arial&quot;;
    padding: 8px , 16px;
    border-radius: 12px;
    border: 3px solid lightblue;        
    letter-spacing: 1.5px;
}

QPushButton:hover {
    background-color: #3a3f5c;
    color: #ffffff;
    border: 2px solid #5ef1f3;
}

QPushButton:pressed {
    background-color: #22263b;
    padding-left: 16px;
    padding-top: 16px;
}
</string>
        </property>
        <property name="text">
         <string>Show mistakes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="showSolution">
        <property name="maximumSize">
         <size>
          <width>3000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    color: #e0e0e0;
    background-color: #2b2f4a;         
    font-size: 17px;
    font-weight: 800;
    font-family: &quot;Orbitron&quot;, &quot;Poppins&quot;, &quot;Segoe UI&quot;, &quot;Arial&quot;;
    padding: 8px , 16px;
    border-radius: 12px;
    border: 3px solid lightblue;        
    letter-spacing: 1.5px;
}

QPushButton:hover {
    background-color: #3a3f5c;
    color: #ffffff;
    border: 2px solid #5ef1f3;
}

QPushButton:pressed {
    background-color: #22263b;
    padding-left: 16px;
    padding-top: 16px;
}</string>
        </property>
        <property name="text">
         <string>Show solution</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="resetCells">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    color: #e0e0e0;
    background-color: #2b2f4a;         
    font-size: 17px;
    font-weight: 800;
    font-family: &quot;Orbitron&quot;, &quot;Poppins&quot;, &quot;Segoe UI&quot;, &quot;Arial&quot;;
    padding: 8px , 16px;
    border-radius: 12px;
    border: 3px solid lightblue;        
    letter-spacing: 1.5px;
}

QPushButton:hover {
    background-color: #3a3f5c;
    color: #ffffff;
    border: 2px solid #5ef1f3;
}

QPushButton:pressed {
    background-color: #22263b;
    padding-left: 16px;
    padding-top: 16px;
}</string>
        </property>
        <property name="text">
         <string>Reset cells</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
